src/shared_modules/utils/zlibHelper.hpp

```diff
@@ -88,6 +92,89 @@ namespace Utils
             } while (len == sizeof(buf));
             outputFile.close();
         }
+
+        /**
+         * @brief Uncompress ZIP file and returns a list with the decompressed files.
+         *
+         * @param zipFilePath Compressed (.zip) file path.
+         * @param outputDir Folder where the output files will be stored.
+         * @return std::vector<std::string> List of decompressed files.
+         */
+        static std::vector<std::string> zipDecompress(const std::filesystem::path& zipFilePath,
+                                                      const std::filesystem::path& outputDir)
+        {
+            // Open .zip file.
+            auto unzFile {unzOpen(zipFilePath.c_str())};
+            if (!unzFile)
+            {
+                throw std::runtime_error {"Unable to open compressed file: " + zipFilePath.string()};
+            }
+
+            // Get .zip file information (amount of files, i.e.).
+            unz_global_info globalInfo;
+            if (unzGetGlobalInfo(unzFile, &globalInfo) != UNZ_OK)
+            {
+                unzClose(unzFile);
+                throw std::runtime_error {"Unable to get global information of file: " + zipFilePath.string()};
+            }
+
+            // Iterate all compressed files within the .zip file.
+            std::vector<std::string> decompressedFiles(globalInfo.number_entry);
```

Comment:
Nitpick: how about 
```suggestion
            std::vector<std::string> decompressedFiles;
            decompressedFiles.reserve(globalInfo.number_entry);
```
And then use `push_back` when inserting each name ?
It doesn't make much of a difference, but I think it conveys the intent better (setting the capacity instead of the size)

Reply:
Done!